metadata/metadata.ts

```diff
@@ -6066,7 +6066,7 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
     image_url: "../public/erc1155/images/487.png",
     attributes: [
       { trait_type: "Tradable", value: "Yes" },
-      { trait_type: "Boost", value: "Resources" },
+      { trait_type: "Boost", value: "Fruit" },
       {
         display_type: "boost_percentage",
         trait_type: "Increase Banana Speed",
```

Comment:
"Decrease growth time"?

Reply:
I didn't notice that, will change it asap